package-lock.json

```diff
@@ -0,0 +1,4318 @@
+{
```

Comment:
Looks like AppAuth-JS uses yarn so we can probably just drop the file to avoid any confusion or conflicts.